www/templates/account/plans/plan-summary.php

```diff
@@ -99,4 +106,15 @@ function error(err) {
             submitButton.removeAttribute('disabled');
         });
     })();
+</script>
+<script>
+    (() => {
+        document.addEventListener('cc-upgrade-error', e => {
+            const el = document.createElement('div');
+            el.classList.add('notification-banner', 'notification-banner__error');
+            el.innerText = e.detail;
+            document.querySelector('#notification-banner-container').appendChild(el);
```

Comment:
This will continue appending new notification divs, right? Is this desirable or we want to clean up the previous error message?

Reply:
We should probably clean it up but I want to get this out and then quickly put together something a bit better